transcendence-app/src/shared/db/models.ts

```diff
@@ -10,11 +10,11 @@ export interface BaseEntity {
 
 export interface Query {
   text: string;
-  values: Array<string | null>;
+  values: any[];
```

Comment:
I think `any[]` is a more suited type for this property because it could represent an array with strings, numbers, Dates, nulls, etc...

For example, if there is an entity like this:

```
{
  id: string
  name: string;
  age: number;
  created_at: Date;
  someOtherProp: null;
  ...
}
```